src/middleware/set-local-vars-middleware.ts

```diff
@@ -3,7 +3,7 @@ import {
   getAccountManagementUrl,
   getAnalyticsCookieDomain,
   getGtmId,
-  getLanguageToggleEnabled
+  getLanguageToggleEnabled,
```

Comment:
Could you squash this change into the previous commit? Generally we don't like having separate linting commits (it means that when you do a git blame, you don't get linked to an explanatory commit message)

Reply:
@BeckaL Is it not possible to select the button "squash and merge" when this PR will be approved or that will not fix your issue?

Reply:
Generally we don't use the "squash and merge" button as we like keeping separate logical commits (if you had multiple logical commits here rather than one, we'd lose those logical commits in the history). It's also good not to have refactoring commits for people reviewing (I review commit by commit so it can get noisy).